server/src/infra/repositories/system-config.repository.ts

```diff
@@ -20,6 +25,11 @@ export class SystemConfigRepository implements ISystemConfigRepository {
     return this.repository.save(items);
   }
 
+  async getLatestAvailableVersion(): Promise<GithubRelease> {
+    const { data } = await axios.get<GithubRelease>('https://api.github.com/repos/immich-app/immich/releases/latest');
+    return data;
+  }
+
```

Comment:
This is perfect adding it to a repository method, but I don't think it should be in system config. I don't think we have an existing repository that is appropriate for this either. Can you add a new one, like `IServerInfoRepository` or `IGitHubRepository`?